### PR TITLE
chore(release): Prep for 1.0.0 release

### PR DESCRIPTION
**Pull request by palpatim**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by royjit** [comment]
AWSCognitoAuthPlugin Info.plist is missing

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 1.0.0 (2020-05-26)
+
+### Bug Fixes
+
+* Tools: Add npx to amplify-app and fix typo - fixes #486 (#490) ([#486](https://github.com/aws-amplify/amplify-ios/issues/486)) ([c7d11a7](https://github.com/aws-amplify/amplify-ios/commit/c7d11a7b1291a2aa588fdcca5bf51e259490d9b5))
```
Comment (royjit) [inline CHANGELOG.md:9]: The numbers are not linked 486, 490

Comment (palpatim) [reply CHANGELOG.md]: Thanks, fixed by linking 486

**Comment by palpatim** [comment]
> AWSCognitoAuthPlugin Info.plist is missing

The Info.plist didn't have a correct version before--looks missed it in RC1, too. Let's remember that next time we rename targets. :(

**Pull request merged**